test/per_file_coverage.sh

```diff
@@ -28,7 +28,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/common/tcp:94.6"
 "source/common/thread:0.0" # Death tests don't report LCOV
 "source/common/tracing:96.1"
-"source/common/upstream:96.0"
+"source/common/upstream:95.8"
```

Comment:
have you tried testing these by static casting an out of bounds number and using EXPECT_ENVOY_BUG?

Reply:
No, I was fixated on the now missing coverage lines in the case that a none-option is returned. But I'll try that and see if that increases the coverage enough to not require this change 👍 .